tests/test_util/aiohttp.py

```diff
@@ -65,9 +67,18 @@ def call_count(self):
         return len(self.mock_calls)
 
     @asyncio.coroutine
-    def match_request(self, method, url, *, auth=None, params=None): \
+    def match_request(self, method, url, *, data=None, auth=None,
+                      params=None): \
             # pylint: disable=unused-variable
         """Match a request against pre-registered requests."""
+        self.last_request = {
```

Comment:
We should replace this with `self.mock_calls.append()` call if we match a request. So we should always register each call. And instead of appending a tuple, let's append the last request dictionary.